recipes/neurokit2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "NeuroKit2" %}
+{% set version = "0.0.42" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
It looks like there is some capitalization issues. In the url, the package name should be `neurokit2`.

Reply:
Fixed